src/item.h

```diff
@@ -1364,6 +1364,11 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
         * Returns label from "item_label" itemvar and quantity
         */
         std::string label( unsigned int quantity = 0 ) const;
+        /**
+         * Returns true if the item has infinite charges
```

Comment:
The comment only repeats part of the function name and does provide no further information. In fact, it does not even tell what it returns (if at all) when the item does not have infinite charges. Functions like this probably don't need a specific comment at all.